README.md

```diff
@@ -1,57 +1,95 @@
 
-2. **Objective**:  
+### Objective
+
 To facilitate setting up dataspaces that follow certain principles, serving as an initial platform for data providers, apps, and data consumers.
 
-3. **Roles**:
-  - Data Providers: Organizations that either offer a data source with raw data or an app with processed data. In all cases, access conditions are set by the data owner.
-  - App Providers: Organizations that act as intermediaries, adding value to raw data. They act as a Data Consumer on behalf of their end users, and as a Data Provider for their end users.
-  - Data Consumers: Organizations that use data via Service Providers or directly.
-  - Dataspace initiators: Organizations that setup and manage the dataspace.
+### Roles
+
+- Data Providers: Organizations that either offer a data source with raw data or an app with processed data. In all cases, access conditions are set by the data owner.
+- App Providers: Organizations that act as intermediaries, adding value to raw data. They act as a Data Consumer on behalf of their end users, and as a Data Provider for their end users.
+- Data Consumers: Organizations that use data via Service Providers or directly.
+- Dataspace initiators: Organizations that setup and manage the dataspace.
 
-4. **Principles**:
-    1. Data owners (issuers) can issue access to their data, even if through federated apps.
-    2. Data stays at its source unless caching or staging is essential.
-    3. Data consumers choose their identity providers.
+### Principles
+
+1. Data owners (issuers) can issue access to their data, even if through federated apps.
+2. Data stays at its source unless caching or staging is essential.
+3. Data consumers choose their identity providers.
+
+### Customer Journeys
 
-5. **Customer Journeys**:  
 The wiki describes the following [Customer Journeys](/Poort8.Dataspace.Wiki/customerjourneys.md) in more detail:
- - Initiating Dataspace Core
- - Onboarding Data Sources
- - Onboarding Data Owners and Consumers 
- - Data Sources Becoming Independent
- - Adding providers and apps
-The first 3 journeys comprise the launch of a first (prototype) of a data space. Journeys 4 and 5 allow data sources and Service Providers to become independent contributors to the data space.
-
-6. **Functionalities**:  
-The Dataspace Core provides services for the organization registry, the organization onboarding process, the data space manager  and  (optionally) an authorization registry. With the dataspace manager the data space standards can be managed, such as the requirements for authentication and onboarding, and the definition of a dataspace data model.
+
+- Initiating Dataspace Core
+- Onboarding Data Sources
+- Onboarding Data Owners and Consumers
+- Data Sources Becoming Independent
+- Adding providers and apps
+  The first 3 journeys comprise the launch of a first (prototype) of a data space. Journeys 4 and 5 allow data sources and Service Providers to become independent contributors to the data space.
```

Comment:
Deze zin staat nog direct naast "Adding providers and apps", volgens mij komt 'ie met een extra enter er wel onder te staan.